### PR TITLE
USWDS-Site: HTML proofer and snyk fixes

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
- Updated 301 redirect links to fix html-proofer failures.
- Updated snyk ignore

## Related issue
N/A

## Preview link

- [Performance glossary](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-html-proofer-dec/performance/glossary/)
- [Performance how](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-html-proofer-dec/performance/how/)
- [Pronoun pattern](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-html-proofer-dec/patterns/create-a-user-profile/pronouns/)

## Problem statement
1. We received new failures related to 301 links in html-proofer:
  ```
For the Links > External check, the following failures were found:

* At ./_site/patterns/create-a-user-profile/pronouns/index.html:2983:

  External link https://www1.nyc.gov/assets/hra/downloads/pdf/services/lgbtqi/Gender%20Pronouns%20final%20draft%2010.23.17.pdf failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/glossary/index.html:3020:

  External link https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/glossary/index.html:3183:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/glossary/index.html:3196:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/glossary/index.html:3203:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/glossary/index.html:3217:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/glossary/index.html:3239:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/how/index.html:3117:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/how/index.html:3127:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/how/index.html:3150:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/how/index.html:3165:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/how/index.html:3176:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/how/index.html:3197:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)

* At ./_site/performance/how/index.html:3353:

  External link https://developers.google.com/web/tools/lighthouse/ failed: got a time out (response code 301) (status code 301)
  ```
2. We received a new snyk failure related to nanoid:
  ```
  Issues with no direct upgrade or patch:
    ✗ Improper Input Validation [Medium Severity][https://security.snyk.io/vuln/SNYK-JS-NANOID-8492085] in nanoid@3.3.7
      introduced by @uswds/compile@1.2.0 > postcss@8.4.40 > nanoid@3.3.7
    This issue was fixed in versions: 3.3.8, 5.0.9
  ```

## Testing and review

- Confirm all updated links point to the expected location.
- Confirm there are no text updates needed for the related url updates
- Confirm no changelogs are needed (no meaningful changes from these url updates)

### CODE REVIEW

File: pages/documentation/guidance/performance/glossary.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The following gives a brief overview of a handful of metrics. If you're looking
 
 ### Onload
 
-[The `load` event](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload) (as shown in MDN Web Docs) fires at the end of the document loading process. At this point, all of the objects in the document are in the DOM, and all the images, scripts, links, and sub-frames have finished loading.
+[The `load` event](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) (as shown in MDN Web Docs) fires at the end of the document loading process. At this point, all of the objects in the document are in the DOM, and all the images, scripts, links, and sub-frames have finished loading.
```
Comment (amyleadem) [inline pages/documentation/guidance/performance/glossary.md:85]: > [!Note]
> This new url has different content than the previous url. Before, it pointed to `onload` whereas the new url points to the `load` event listener.
>
> [Before](https://web.archive.org/web/20220317225305/https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload) | [After](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event)

File: pages/documentation/guidance/performance/glossary.md [hunk 2]
```diff
@@ -389,6 +389,6 @@ var numberOfElements = document.getElementsByTagName('*').length;
 [critical CSS]: https://www.smashingmagazine.com/2015/08/understanding-critical-css/
 [WebPagetest]: https://www.webpagetest.org/
 [SpeedCurve]: https://speedcurve.com/
-[Lighthouse]: https://developers.google.com/web/tools/lighthouse/
+[Lighthouse]: https://developer.chrome.com/docs/lighthouse/overview/
```
Comment (amyleadem) [inline pages/documentation/guidance/performance/glossary.md:392]: > [!note]
> The content in the new url appears to be the same as the previous url
> 
> [Before](https://web.archive.org/web/20220102193923/https://developers.google.com/web/tools/lighthouse/) | [After](https://developer.chrome.com/docs/lighthouse/overview/)

File: .snyk
Comment (amyleadem) [inline .snyk:1]: > [!note]
> nanoid is the only new snyk failure for this PR. However, I also updated inflight (which was recently updated in PR #2995) to keep the snyk updates in sync with each other.

**Review by mahoneycm — DISMISSED** [review]
LGTM! New links look appropriate and tests are passing

**Review by annepetersen — DISMISSED** [review]
LGTM

**Comment by amyleadem** [comment]
@annepetersen fixed one more 301 failure 🙃 ... should be good to merge now

**Review by annepetersen — APPROVED** [review]
trying again!

**Pull request merged**